examples/graphcool-demo/src/reviews/reviewSaga.js

```diff
@@ -9,22 +9,40 @@ import {
 } from './reviewActions';
 
 export default function* reviewSaga() {
-    yield [
-        takeEvery(REVIEW_APPROVE_SUCCESS, function* () {
-            yield put(showNotification('resources.reviews.notification.approved_success'));
-            yield put(push('/reviews'));
+    yield all([
+        takeEvery(REVIEW_APPROVE_SUCCESS, function*() {
+            yield put(
+                showNotification(
```

Comment:
I suggest you use the new `onSuccess` meta in the action creator for that purpose (in a future PR).